eqt/ui/FormDialog.py

```diff
@@ -225,9 +238,9 @@ def saveAllWidgetStates(self):
         '''
         self.formWidget.saveAllWidgetStates()
 
-    def getWidgetStates(self):
+    def getSavedWidgetStates(self):
```

Comment:
add to the CHANGELOG, and add an alias with deprecation as `getWidgetStates`

Reply:
done